datalad/interface/base.py

```diff
@@ -651,7 +651,7 @@ class Interface(object):
 
     _OLDSTYLE_COMMANDS = (
         'AddArchiveContent', 'CrawlInit', 'Crawl', 'CreateSiblingGithub',
-        'CreateTestDataset', 'Export', 'Ls', 'SSHRun', 'Test')
+        'CreateTestDataset', 'Export', 'Ls', 'SSHRun', 'ShellCompletion', 'Test')
```

Comment:
This tuple should shrink and never grow. It is fine for me to have outlier commands like this, but they should be treated as their own special case further down, and not populate a TODO list.

Unless, I got it wrong, and this new command is actually meant to be RF'ed into a normal datalad command.

Reply:
To say the truth I forgot the differences between plugins, "oldstyle" commands" and "normal datalad commands".  What I know is that this command output to stdout should be the only thing produced to the stdout.  I do not see a point of knowing the `json` representation etc.  But I guess it could still be just for tailored renderer to "present" it, but that just makes it like a "hello world" in java ;)

Reply:
That is my point. Why make this new command an old-style command, when it is not even completely understood what that means. As stated above, I am not concerned about the behavior of the command, only about how it is injected into the API.

Reply:
ok, now it is fancy new, but I followed up with #5575 for consideration